Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/MagDurCurves/CurveForm.tsx

```diff
@@ -230,9 +236,9 @@ export default function CurveForm(props: IProps) {
                         zoom={true} pan={false}
                         useMetricFactors={false} XAxisType={'log'}
                         >
-                        <Line highlightHover={false} showPoints={true} lineStyle={'-'}
+                        <Line highlightHover={false} showPoints={false} lineStyle={'-'}
                             color={"#A30000"} data={curve} />
-                        
+                        {curve.slice(0, -1).map((p) => (<Circle data={[p[0], p[1]]} color={"#FFFFFF"} radius={8} borderColor={"#A30000"} borderThickness={2} text={String(curve.indexOf(p) + 1)} />))}
```

Comment:
turns out this is a bit of a performance drag.
`curve` is an array that stays the same between renders (unless you change it on purpose), but `curve.slice` get's recomputed every time and results in a complete re-render of the plot. (Technically it's the reference to the array that matters. `curve` stays the same but `curve.slice(...)` is a new reference. I would swap this to ` curve.map((p,i) => (i== curve.length? null : <circle...../>)`